features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```
@@ -1519,6 +1519,22 @@
   ],
   "console.user_roles.scopes.update": ["internal_role_mgt_update"],
   "console.user_roles.scopes.delete": ["internal_role_mgt_delete"],
+  "console.entitlement.enabled": true,
+  "console.entitlement.feature": [
+    "console.roles"
```

Shouldn't this be as follows?

```suggestion
    "console:roles"
```